marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/DhsDeployer.java

```diff
@@ -99,20 +90,24 @@ protected void prepareAppConfigForDeployingToDhs(HubConfig hubConfig) {
         removeHubInternalConfigFromConfigDirs(appConfig);
         addEntityConfigToConfigDirs(hubConfig.getHubProject(), appConfig);
 
-        // 8000 is not available in DHS
-        int port = hubConfig.getPort(DatabaseKind.STAGING);
-        logger.info("Setting App-Services port to: " + port);
-        appConfig.setAppServicesPort(port);
+        if(hubConfig.getIsProvisionedEnvironment()){
```

Comment:
For testing this, you can add a scenario to DeployAsDeveloperTest (which is currently failing) where this is false, and you call prepareAppConfig and then verify that the below modifications were not made.

Reply:
I've added a test for on prem (vanilla datahub)